scenarios/test_codegen/test/integration-raw-events-test.ts

```diff
@@ -23,7 +23,7 @@ import {
 // import { setLogLevel } from "generated/src/Logging.bs.js";
 // require("mocha-reporter").hook(); //Outputs filename in error logs with mocha-reporter
 
-describe("Raw Events Integration", () => {
+describe.skip("Raw Events Integration", () => {
```

Comment:
I couldn't make the test work, because hardhat doesn't actually run an rpc server.

I think it's fine to disable it for now, since our integration tests do pretty much the same thing.

Reply:
Do we have an rpc integration test? Agree this test is fragile and I'd like to remove hardhat from our tests

Reply:
I think no. Need to make one of the integration tests to use RPC